### PR TITLE
PreRender.c retail 1 not matching

### PR DESCRIPTION
**Pull request by shawlucas**
Not sure if this should be submitted but I'll try anyway. Most of the other functions were figured out from using MM decomp but the DivotFilter function is complicated and above my knowledge, couldn't figure it out.

### CODE REVIEW

**Review by fig02 — APPROVED** [review]
partial files are fine. feel free to post a scratch in the discord for the function gving you a problem

**Review by cadmic — APPROVED** [review]

**Review by Dragorn421 — APPROVED** [review]

File: src/code/PreRender.c [hunk 1]
```diff
@@ -545,7 +549,7 @@ void PreRender_AntiAliasFilter(PreRender* this, s32 x, s32 y) {
     s32 buffB[5 * 3];
     s32 xi;
     s32 yi;
-    s32 pad;
+    s32 temp;
```
Comment (Thar0) [inline src/code/PreRender.c:552]:
```suggestion
    s32 invCvg;
```

**Review by Thar0 — COMMENTED** [review]
Temp var name suggestion

File: src/code/PreRender.c (hunk 1 above)
Comment (Dragorn421) [reply src/code/PreRender.c:552]: inv is 1/ though I get what you mean

Comment (Thar0) [reply src/code/PreRender.c:552]: It's an "additive inverse" which should be clear from the context

**Pull request merged**